internal/utils/stat.go

```diff
@@ -76,3 +76,37 @@ func Stat(ctx context.Context, store *multistore.Store, root cid.Cid, sel ipld.N
 	})
 	return res, nil
 }
+
+func MapKeys(ctx context.Context, root cid.Cid, loader ipld.Loader) ([][]byte, error) {
```

Comment:
Can you add a comment describing what that function does? Just for future documentation